.github/pr-body.md

```diff
@@ -1,3 +1,5 @@
+!!! ONLY A TEST !!!
+
 Created by Mondoo Tools via GitHub Actions
```

Comment:
I do not understand why we need to merge the test. Can we not trigger this without merging to main?

Reply:
I don't think so.

I need additional output in the `pr-test-lint.yaml` which is triggered by the GHA PR. And I also need the trigger from GHA, otherwise actor and some other things would be different, which I suspect to be part of the problem.